transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollSocketTcpMd5Test.java

```diff
@@ -54,7 +54,7 @@ public void setup() {
         server = (EpollServerSocketChannel) bootstrap.group(GROUP)
                 .channel(EpollServerSocketChannel.class)
                 .handler(new ChannelInboundHandlerAdapter())
-                .bind(new InetSocketAddress(0)).syncUninterruptibly().channel();
+                .bind(new InetSocketAddress(NetUtil.LOCALHOST4, 0)).syncUninterruptibly().channel();
```

Comment:
I know what you are asking but `NetUtil.LOCALHOST`may be an ipv6 address. We need to ensure we always use an ipv4 one.